### PR TITLE
build: remove webpack

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

remove webpack since it's unused for now

## Testing

CI

### CODE REVIEW

File: packages/browser/package.json [hunk 1]
```diff
@@ -14,10 +14,8 @@
     "precommit": "lint-staged",
     "build": "rm -rf lib/ && tsc -p tsconfig.build.json",
     "lint": "eslint --ext .ts src",
-    "package": "pnpm build && webpack",
     "test": "jest",
     "test:coverage": "jest --silent --coverage",
-    "report": "WITH_REPORT=true pnpm package",
```
Comment (demonzoo) [inline packages/browser/package.json:20]: Is this relevant? We can still keep `report` by calling `pnpm build` instead

Comment (gao-sun) [reply packages/browser/package.json:20]: report is a webpack thing

Comment (gao-sun) [reply packages/browser/package.json:20]: `WITH_REPORT` is just for webpack

Comment (demonzoo) [reply packages/browser/package.json:20]: Oops, I didn't know that. OK then

**Review by demonzoo — APPROVED** [review]

**Pull request merged**